### DISCUSSION TITLE
Is it possible to only apply night mode on certain layouts?

### DISCUSSION THREAD

**Discussion by daydream05** [system-ui/theme-ui#1190] [General]
I have docs subdirectory on my site and I'd like it to be the only one affected when I use dark/night mode while the rest of the site stays the same. Is this possible? 

For context, I use gatsby

**Comment by hasparus** [comment]
Take a look at your React tree in the devtools. There's a _ColorModeProvider_ there, which keeps the state of current color mode.

The _ThemeProvider_ your import from `theme-ui` already has the whole deal with _ColorModeProvider_ and _MdxProvider_,
but you can use the _ThemeProvider_ from `@theme-ui/core` at the root of your site, and include _ColorModeProvider_ somewhere lower in the tree.
